src/layers/fp64/scatterplot-layer/scatterplot-layer.js

```diff
@@ -143,6 +146,7 @@ export default class ScatterplotLayer extends BaseLayer {
       [value[i + 0], value[i + 1]] = this.df64ify(position[2]);
       i += size;
     }
+    console.log("height: ", value);
```

Comment:
not sure if missed some previous comments, but shall we remove the console.logs here and below?